kqueen_ui/server.py

```diff
@@ -6,10 +6,13 @@
 from kqueen_ui.blueprints.ui.views import ui
 from kqueen_ui.exceptions import KQueenAPIException
 from kqueen_ui.utils.filters import filters, context_processors
+from kqueen_ui.utils.loggers import setup_logging
 
 import logging
 
-logger = logging.getLogger(__name__)
+# Logging configuration
+setup_logging('/code/kqueen_ui/utils/logger_config.yml')
```

Comment:
This path can't be hardcoded, absolute path /code is specific for container deployment

Reply:
yep, already work on it, thx